lib/g_searcher/accounts.ex

```diff
@@ -0,0 +1,10 @@
+defmodule GSearcher.Accounts do
+  import Ecto.Query, warn: false
```

Comment:
I think we don't need this one for now, Am I correct?

Reply:
True, I don't even remember putting this here 🤔